WADSuite/Button.cs

```diff
@@ -0,0 +1,127 @@
+﻿//******************************************************************************
```

Comment:
Instead of "WADSuite" as the folder for all of these tests, could we update it to be "IntegrationTests" or "AutomatedTests" or something? I'm not sure if folks will know what WAD means.

Reply:
I've renamed them to UITests. Let me know if you think XAMLUITests sounds better, or if you have other recommendations. Thanks!